beacon/beacon-basic-string.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>W3C Beacon Basic String Test</title>
+    <meta name="timeout" content="long">
+    <meta name="author" title="Microsoft Edge" href="https://www.microsoft.com">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <script src="/common/utils.js"></script>
+    <script src="beacon-common.js?pipe=sub"></script>
+    <script>
+        "use strict";
+        runTests(stringTests);
```

Comment:
I'll will investigate on WebKit side. The test takes up to ~130 seconds on some of our debug bots.
In any case, feel free to land as is. It could simply be an issue with our implementation as it is very new.